nncf/experimental/common/pruning/operations.py

```diff
@@ -248,7 +254,7 @@ def mask_propagation(
         else:
             # Each branch/mask should have a single group along the same dimension. These groups are joined, all others
             # are invalidated.
-            output_mask = PropagationMask()
+            dim_groups_map = {}
             first_mask = input_masks[0]
             dims_to_invalidate = []
             for dim in first_mask.dim_groups_map:
```

Comment:
The issue happened here. ElementWise operation gets 2 masks - one is "incorrectly" empty (with empty `dim_groups_map` dict), another - the mask that should be invalidated.
This loop did nothing, since the first mask is empty and the second mask was not invalidated. As a result, some group may have no consumer and can be valid. 
@yujiepan-work @daniil-lyakhov